sunfish.go

```diff
@@ -25,7 +25,7 @@ func NewSunfish() *Sunfish {
 	var err error
 	sf.DBSession, err = mgo.Dial("localhost")
 	if err != nil {
-		fmt.Println("Could not reach a Mongo server. Make you have Mongo configured correctly.")
+		fmt.Println("Could not reach a Mongo server. Ensure Mongo is configured correctly.")
```

Comment:
Errors are handle differently with one of the outstanding pr's i have. This won't be mergeable